docs/products/kafka/howto/rotate-credentials.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Rotate credentials for an Apache Kafka subscription
+sidebar_label: Rotate credentials
+---
+
+import ConsoleLabel from "@site/src/components/ConsoleIcons";
+import RelatedPages from "@site/src/components/RelatedPages";
+
+Rotate credentials for an Apache Kafka subscription to replace outdated credentials and keep access secure and approved.
+The request must be approved by another member of the owner group before new
+credentials can be downloaded.
```

Comment:
It might be redundant here, especially in the into statement? Move it to **Limitations** section? See my comment below.

Reply:
See https://github.com/aiven/aiven-docs/pull/798/files#r2063526579